asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/ConfluenceRestClient.java

```diff
@@ -162,6 +163,16 @@ public void addAttachment(String contentId, String attachmentFileName, InputStre
         });
     }
 
+    @Override
+    public void addAttachment(String contentId, String attachmentFileName, Path attachmentContent) {
+        HttpPost addAttachmentRequest = this.httpRequestFactory.addAttachmentRequest(contentId, attachmentFileName, attachmentContent);
+        sendRequestAndFailIfNot20x(addAttachmentRequest, (response) -> {
+            //closeInputStream(attachmentContent);
```

Comment:
Why add commented out code here? If it needs to stay in, then I suggest adding an explanatory comment with it.